src/backend/aspen/app/app.py

```diff
@@ -22,24 +23,28 @@
 else:
     aspen_config = DockerComposeConfig()
 
+deployment = os.getenv("DEPLOYMENT_STAGE")
+
 # We should be able to allow this in all environments and only alert on prod.
 # Init as early as possible to catch more
 sentry_sdk.init(
-    aspen_config.SENTRY_URL,
+    dsn=aspen_config.SENTRY_URL,
+    integrations=[FlaskIntegration()],
```

Comment:
I think we want to set the environment here? (see https://docs.sentry.io/platforms/python/configuration/environments/)